src/components/DonationButtons/index.ts

```diff
@@ -0,0 +1 @@
+export { default } from './DonationButtons';
```

Comment:
Does this need to be a separate file? Can you just export from DonationsButton.tsx directly?

Reply:
Deleted the file.